distributions/atalanta-rtic/pcs-pass/src/lib.rs

```diff
@@ -13,8 +11,10 @@ use syn::ItemMod;
 
 pub const PCS_ATTR_IDENT: &str = "fast";
 
-// HACK: pass the list of dispatchers from PCS pass to core pass backend.
-thread_local!(pub static PCS_DISPATCHERS: RefCell<Vec<proc_macro2::Ident>> = const { RefCell::new(vec![]) });
+pub struct PcsPassArtifacts {
+    // the list of dispatchers from PCS pass to core pass backend.
+    pub pcs_dispatchers: Vec<proc_macro2::Ident>,
+}
```

Comment:
I guess these are commonly called "Parameters" or "Params". `pcs_pass::CoreParams`  seems like the most ideal name to me.